src/ros_can_nodes/include/ros_node_lib/RosNode.hpp

```diff
@@ -0,0 +1,208 @@
+/*
+ * Date Started:
+ * Original Author: Nuno Das Neves
+ * Editors: Yiwei Han
+ * ROS Node Name:
+ * ROS Package: ros_can_nodes
+ * Purpose:
+ * This code is released under the MIT  License. Copyright BLUEsat UNSW, 2018
```

Comment:
```suggestion
 * This code is released under the BSB License. Copyright BLUEsat UNSW, 2018
```
(All the code in this repo is under bsb because it was forked from roscpp)